app/frontend/components/shared/chefs/styles.scss

```diff
@@ -0,0 +1,400 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900");
+@import url("https://fonts.googleapis.com/css?family=Material+Icons");
+@import "bootstrap/dist/css/bootstrap.min.css";
+@import "formiojs/dist/formio.form.css";
+
+// Variables
+$bcgov-blue: #003366;
+$bcgov-blue-secondary: #38598a;
+$bcgov-orange: #fcba19;
+$bcgov-grey: #efefef;
+$bcgov-font: #313132;
+$bcgov-error: #d8292f;
+$bcgov-link: #1a5a96;
+$bcgov-link-hover: #3b99fc;
+$bcgov-input-placeholder: #606060;
+$bcgov-font-error: #a12622;
+$bcgov-font-warning: #6c4a00;
+$bcgov-font-success: #2d4821;
+$bcgov-alert-error: #f2dede;
+$bcgov-alert-warning: #f9f1c6;
+$bcgov-alert-info: #d9eaf7;
+$bcgov-alert-success: #dff0d8;
+
+// Typography
+.v-application {
+  font-family: -apple-system, BlinkMacSystemFont, BCSans, Roboto, Verdana, Arial, sans-serif !important;
+  line-height: 1.4;
+  font-size: 0.875rem;
+}
+
+h1 {
+  font-size: 1.9em;
+}
+h2 {
+  font-size: 1.7em;
+}
+h3 {
+  font-size: 1.3em;
+}
+h4 {
+  font-size: 1.05em;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  color: $bcgov-font;
+  line-height: 1.2;
+  font-weight: bold;
+}
+
+p {
+  color: $bcgov-font;
+  line-height: 1.7;
+}
+
+// links
+a {
+  color: $bcgov-link;
+  &:hover {
+    color: $bcgov-link-hover;
```

Comment:
@sunto Github doesn't support scroll on comment selection so this is for line 25 to 154. I think these styles can be stripped out as they are for their application. I think we need only the formio overrides. It's also because the h1 and a tags etc is going to override our default styles

Reply:
alternatively can wrap these inside the .form-wrapper class. That being said we should strip the styles that we don't need